src/trpc/router/userRouter.ts

```diff
@@ -103,16 +103,18 @@ export const userRouter = router({
     editMailingSettings: protectedProcedure.input(editUserMailingSettingsSchema).mutation(async ({ input, ctx }) => {
         accessCheck(checkRoleForAccess(ctx.session.user.role, 'editUser'));
         const result = await userMethods.editMailingSettings(input);
-        await historyEventMethods.create({ user: ctx.session.user.id }, 'editUserMailingSettings', {
-            groupId: undefined,
-            userId: result.userId,
-            before: undefined,
-            after: {
-                type: input.type,
-                value: result[input.type],
-                organizationUnitId: result.organizationUnitId || undefined,
-            },
-        });
+
+        result.userId &&
```

Comment:
Don't just stop saving logs, add new type of event for new action. Maybe mark old one as deprecated

Reply:
old one needed too: userId is optional now in mailSettings, but I'll watch how remove this condition